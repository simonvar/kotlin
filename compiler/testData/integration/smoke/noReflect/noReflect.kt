package noReflect

fun main() {
    String::class.annotations
}