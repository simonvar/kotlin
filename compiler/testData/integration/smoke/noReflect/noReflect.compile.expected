ERR:
noReflect.kt:4:19: warning: call uses reflection API which is not found in compilation classpath. Make sure you have kotlin-reflect.jar in the classpath
    String::class.annotations
                  ^

Return code: 0
